test/test_scanner.py

```diff
@@ -240,7 +241,7 @@ def condensed_filepath(self, url, package_name):
             for d in list_data
         ),
     )
-    @unittest.skipUnless(LONG_TESTS() > 0, "Skipping long tests")
+    @unittest.skipUnless(LONG_TESTS(), "Skipping long tests")
```

Comment:
```suggestion
    @pytest.mark.skipif(not LONG_TESTS, reason="Skipping long tests")
```

Reply:
@metabiswadeep, what about this suggestion?

Reply:
Missed it somehow, will implement it now.